_rules/non-visual-location-reference-825ule.md

```diff
@@ -0,0 +1,506 @@
+			</li>
+			<li>
+				<a href="https://www.w3.org/participate/">Participate</a>
+			</li>
+			<li>
+				<a href="https://www.w3.org/Consortium/membership">Membership</a>
+			</li>
+		</ul>
+	</nav>
+	<div class="col-container">
+		<div class="col">
+			<p>Find the navigation on the right, for the non-essential links</p>
+		</div>
+		<nav>
+			<div class="col">
+				<ul>
+					<li>
+						<a href="https://www.w3.org/Consortium/contact">Contact</a>
+					</li>
+					<li>
+						<a href="https://www.w3.org/Help/">Help and FAQ</a>
+					</li>
+					<li>
+						<a href="https://www.w3.org/Consortium/sponsor/">Donate</a>
+					</li>
+					<li>
+						<a href="https://www.w3.org/Consortium/siteindex">Sitemap</a>
+					</li>
+				</ul>
+			</div>
+		</nav>
+	</div>
+</body>
+```
+
+#### Failed Example 3
+
+The content in the column on the right is indicated with the word "box" (which is a shape indicator based on visual perception) and is also indicated by referencing the word "howdy", but the word "howdy" is in a different paragraph than the word "box".
+
+```html
+<head>
+	<style>
+	.col-container {
+		display: table;
+		width: 100%;
+	}
+	.col {
+		display: table-cell;
+		padding: 16px;
+	}
+	button {
+		background: #FAA;
+	}
+	</style>
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Click the box on the right, for a surprise</p>
+			<p>If you can't find the box here is a hint: it says "Howdy"!</p>
+		</div>
+		<div class="col">
+			<button onclick="alert('Surprise!')">Howdy</button>
+		</div>
+	</div>
+</body>
+```
+
+#### Failed Example 4
+
+This document is using the word "triangle" but in this case the triangle menu is on a different page.
+
+```html
+<body>
+	<p>On the <a href="/test-assets/SC1.3.3-triangle-menu-without-heading.html">information page</a> you can find more examples within the triangle menu</p>
+</body>
+```
+
+#### Failed Example 5
+
+This document is using the word "star" but in this case the star is in an iframe.
+
+```html
+<body>
+<p>More examples can be found when you look underneath the star or underneath the heading examples</p>
+<iframe src="/test-assets/SC1.3.3-star-without-heading.html"></iframe>
+</body>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+The content in the column on the right is indicated with the word "button" which is not a visual reference word.
+
+```html
+<head>
+	<style>
+	.col-container {
+		display: table;
+		width: 100%;
+	}
+	.col {
+		display: table-cell;
+		padding: 16px;
+	}
+	button {
+		background: #FAA;
+	}
+	</style>
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Click the button on the right, for a surprise</p>
+		</div>
+		<div class="col">
+			<button onclick="alert('Surprise!')">Howdy</button>
+		</div>
+	</div>
+</body>
+```
+
+#### Inapplicable Example 2
+
+The content in the column on the right is indicated with the word "box", but this indication is hidden with display:none.
```

Comment:
`display:none` must be in backticks.

"hidden" => both not visible and not in the accessibility tree (since one is enough to make the rule applicable).